src/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -124,6 +124,44 @@ boost::gil::rgba8_image_t::view_t create_starter_image_view(flex_image &object_i
   return starter_image_view;
 }
 
+std::pair<flex_image,flex_dict> create_synthetic_image_from_background_and_starter(
+  flex_image starter,
+  const flex_image &background,
+  std::string label,
+  size_t seed,
+  size_t row_number) {
```

Comment:
The indentation of these arguments, at the same level as the first statement in the body, hides the beginning of the body.

Could we just use `git-clang-format` to normalize the formatting here?

Reply:
Done, thanks for the tip!